src/aria/templates/ClassGenerator.js

```diff
@@ -133,20 +136,25 @@ module.exports = Aria.classDefinition({
                 });
                 return;
             }
-            this.__buildClass(tree, allDeps, callback, context, debug, skipLogError);
+            this.__buildClass(tree, options, callback);
         },
 
         /**
          * Parse a debug template from the tree
          * @param {aria.templates.TreeBeans:Root} tree tree returned by the parser.
-         * @param {Boolean} allDeps If true, all dependencies should be included in the generated class. Otherwise only
-         * put classes which are not currently loaded in the $dependencies section.
+         * @param {aria.templates.CfgBeans:ClassGeneratorCfg} options Options for class generation.
          * @param {aria.core.CfgBeans:Callback} callback callback the callback description
          * @param {Object} context - passes template context information to improve debugging
          * @param {Boolean} debug debug mode
```

Comment:
I think you forgot to remove documentation about the deprecated parameters `context` and `debug`.